setup.py

```diff
@@ -39,4 +58,6 @@
             'Framework :: Django',
             'Framework :: Django :: 2.2',
         ],
+        extras_require=extras,
+        python_requires=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*",
```

Comment:
@odeke-em Is this library compatible with 2.7?

Reply:
Updated to >=3.5.